plugins/modules/aci_l4l7_device.py

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l4l7_device
+short_description: Manage L4-L7 Devices (vns:LDevVip, vns:RsALDevToPhysDomP)
+description:
+- Manage L4-L7 Devices.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  device:
+    description:
+    - Name of L4-L7 device
+    type: str
+    aliases: [ device_name, logical_device_name ]
+  context_aware:
+    description:
+      - Is device Single or Multi context aware
+    type: str
+    choices: [ multi, single ]
+  dev_type:
+    description:
+    - Device type
+    type: str
+    choices: [ physical, virtual ]
+  func_type:
+    description:
+    - Function type of the device
+    type: str
+    choices: [ None, GoTo, GoThrough, L1, L2 ]
+  is_copy:
```

Comment:
Copy is initiated in UI from a separate input, did you investigate all behaviour differences when copy is created? Like are there any configuration dependencies rules?

Reply:
I've gone by what is in the managed object model. `isCopy` is a configurable property of the vns:LDevVIP object, so I have included it in the module.

Reply:
For now, I'll remove isCopy from this module. We can look at updating or creating a separate module for copy as a separate PR if it is required.